src/main/java/net/comments/Test1.java

```diff
@@ -0,0 +1,58 @@
+package net.comments;
+
+import org.hamcrest.MatcherAssert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class Test1 {
+    private final WebDriver[] driver = new WebDriver[1];
+
+    @Test
+    public void test() {
+        final String page = this.driver().getPageSource();
+        String commentsText = "Test text";
+        String number = "31";
+        String savedNumber, savedCommentsText, inactivationSign, categoryName;
```

Comment:
Don't use batch assignment. Create a variable when you are going to use it. For instance, in line 31.